### ISSUE TITLE
Scraping Event Info

### ISSUE THREAD

**Issue by isilrhun** [bugatsinho/bugatsinho.github.io#96]
Great work. Thank you for this awesome addon.
Just wondering wouldn't it be great if the addon could scrape data, images, etc. from TheSportsDB.
There's a working Kodi scraper already.

I think that would take the addon to a whole different level.

**Comment by bugatsinho** [comment]
for what reason would be great idea?

**Comment by isilrhun** [comment]
Well, for one, it would scrape team logos , official match thumbnail or poster, competition badges, etc. so the thumbnails on Kodi are not generic fanarts.
That's the basics. It could also potentially get data like where the location is and other match information, but I do understand that's a lot of work. It's just an idea.

**Comment by isilrhun** [comment]
Something like this.
![screenshot00000Mockup](https://user-images.githubusercontent.com/82736212/150874968-abc6d2a0-bc12-4d65-8731-8f579b90e2b2.png)

**Comment by bugatsinho** [comment]
so do you have the code written for something like that? can you share it here plz. i will try to add it when i will have time!! thank you

**Comment by bugatsinho** [comment]
do you use the api of the website? https://www.thesportsdb.com/api.php

**Comment by isilrhun** [comment]
Ahh man I wish. No I'm a graphic designer and this is just a mock-up. I'd be happy to help with graphics but I definitely don't know enough coding to pull this off

**Comment by bugatsinho** [comment]
how did you add the data on the print you sent?

**Comment by isilrhun** [comment]
Photoshop. I am trying to see if the Kodi scraper would work to pull the data. If I can get it to work, maybe you can incorperate that into the addon? Kinda like seren pulls data from TVDB or something

**Comment by bugatsinho** [comment]
problem with that is that you have to use the api of the website and also become patreon to have access on the event info
![image](https://user-images.githubusercontent.com/18536253/150882853-92b6260c-364b-4977-ac46-0fa2964bd684.png)

**Comment by bugatsinho** [comment]
found this module and i will try to add it but it is only for kodi18 https://github.com/enen92/script.module.thesportsdb

**Comment by isilrhun** [comment]
Yeah, I was looking at that and it looks like it does pull enough data with a free API. Anything more would probably be an overkill for Streaming Addon for Kodi.
Thanks for working on this man. Let me know if you ever need help with Graphix and stuff.

**Comment by bugatsinho** [comment]
only issue is the api 
![image](https://user-images.githubusercontent.com/18536253/150901574-9e2dc2db-477a-4e44-92b2-a6cd530b4eee.png)

**Comment by isilrhun** [comment]
Use "1"
